docusaurus/docs/getting-started/what-s-included/file-loaders.md

```diff
@@ -0,0 +1,20 @@
+---
+id: file-loaders
+title: File loaders for most common files
+sidebar_label: File loaders for most common files
+---
+
+The boilerplate includes [`@moxy/next-common-files`](https://github.com/moxystudio/next-common-files), a Next.js plugin that exports webpack loader rules for common files like images and fonts.
+
+This plugin comes with options that give the developer some choice about how their files are loaded, and this boilerplate uses certain **filename conventions** to specify how certain files are loaded into the web application.
+We implement these filename conventions through suffixes that we add to the filename itself.
+
+Here's a list of all options:
+
+- `.data-url.`: This suffix is used when you want a file to be translated into base64 and sent with your bundle instead of being loaded with a standard URL (e.g., for assets above the fold, you might want to use this to avoid [flashing of unstyled content](https://en.wikipedia.org/wiki/Flash_of_unstyled_content)).
```

Comment:
It should also be noticed that it increases the size of non 6-bits encoded formats in 33%. (https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding#Encoded_size_increase)

Reply:
Added a note.